Source/WebCore/Modules/mediastream/PeerConnectionBackend.h

```diff
@@ -168,6 +169,8 @@ class PeerConnectionBackend
     uint64_t logIdentifier() const final { return m_logIdentifier; }
     ASCIILiteral logClassName() const override { return "PeerConnectionBackend"_s; }
     WTFLogChannel& logChannel() const final;
+    void didLogMessage(const WTFLogChannel&, WTFLogLevel, Vector<JSONLogValue>&&) final { }
+    bool handleLogMessage(const WTFLogChannel&, WTFLogLevel, Vector<JSONLogValue>&&) final;
```

Comment:
Will an observer implement both methods?
Maybe this should be a different observer class?
The benefit would be that, in most cases, the additional observer map would be empty and this would reduce the runtime cost.

Reply:
The default impl of `handleLogMessage()` returns false, i don't think we need a new observer only for this.